akka-docs/src/main/paradox/stream/stream-customize.md

```diff
@@ -484,7 +484,7 @@ or the downstreams. Even for stages that do not complete or fail in this manner,
 
 @@@ div { .group-scala }
 
-## Extending Flow Combinators with Custom Operators
+## Extending Flow operators with Custom Operators
```

Comment:
capital O, or rewrite all headers in this page to lower, which is our recommended style

Reply:
Ah thanks, so `Extending Flow operators with custom operators`?

Reply:
Made it `## Extending Flow Operators with Custom Operators
`